packages/mermaid/src/diagrams/info/infoParser.ts

```diff
@@ -0,0 +1,21 @@
+import type { Info } from 'mermaid-parser';
+import { parse } from 'mermaid-parser';
+
+import { log } from '../../logger.js';
+import type { ParserDefinition } from '../../diagram-api/types.js';
+import type { InfoDB } from './infoTypes.js';
+import { db } from './infoDb.js';
+import { populateCommonDb } from '../common/populateCommonDb.js';
+
+function populateDb(ast: Info, db: InfoDB) {
+  populateCommonDb(ast, db);
+  ast.version && db.setVersion(ast.version);
+}
+
+export const parser: ParserDefinition = {
+  parse: (input: string): void => {
+    const result = parse<Info>('info', input);
+    log.debug(result);
+    populateDb(result, db);
```

Comment:
@Yokozuna59 I had to do this to fix a TS error. Are you not getting that error?

<img width="346" alt="image" src="https://github.com/mermaid-js/mermaid/assets/10703445/e12034ee-6e81-4853-8682-0f8cfafbe8b2">

Reply:
Maybe you haven't rebuilt the parser, because the return value from exported `parse` function return the AST only.

Maybe we should rename the variable to `ast` instead of `result`? Since `result` could indicate the type of `ParseResult<T>`, whereas `ast` should more specific.

Reply:
Yeah, you can rename it. 
The issue is fixed in the latest version, maybe after the rebuild. But I did try rebuilding and installing few times earlier.